example/main.go

```diff
@@ -5,10 +5,11 @@ import (
 	"errors"
 	"fmt"
 	"os"
+
+	"github.com/1password/onepassword-sdk-go"
 )
 
 // [developer-docs.sdk.go.sdk-import]-start
-import "github.com/1password/onepassword-sdk-go"
```

Comment:
gofmt always likes to put this inside of the big import block but for our docs to generate correctly, has to be inbetween the start and stop. I believe you have to manually fix this :)

Reply:
done